### ISSUE TITLE
zkfs pullulation when namenode fail

### ISSUE THREAD

**Issue by crakjie** [mesosphere/hdfs#248]
When a namenode fails the zkfs are not killed
Or when new name node pop it' triggers a new zkfs without checking if another zkfs already exists.

![zkfcpululation](https://cloud.githubusercontent.com/assets/6709653/12177119/e124f200-b56a-11e5-8fb9-29a85915a03b.jpg)

**Comment by asdfsx** [comment]
check the log in sandbox, may be it has failed.

**Comment by crakjie** [comment]
Yes the name node has failed several times, do you mean to check the zkfc logs?
( I'm not able to do it right now since I corrected the name node failure )
